src/snowflake/snowpark/_internal/packaging_utils.py

```diff
@@ -267,24 +276,27 @@ def identify_supported_packages(
                 or package_version_required in valid_packages[package_name]
             ):
                 supported_dependencies.append(package)
+                _logger.info(
+                    f"Package {package_name}{version_text} is supported in Snowflake! The package will not be uploaded."
```

Comment:
By "supported", do you mean "included" or "available"? "Supported" makes it sound as if Snowflake supports use of the package, not necessarily that the package is already present.